Src/Screens/LoggedInScreen/LoggedIn.swift

```diff
@@ -56,13 +56,13 @@ extension LoggedIn: Presentable {
             profilePresentation
         )
         
-        let appVersion = Bundle.main.infoDictionary!["CFBundleShortVersionString"] as? String ?? "0.0.0"
-        let lastNewsSeen = UserDefaults.standard.string(forKey: WhatsNew.lastNewsSeenKey) ?? "0.0.0"
+        let appVersion = Bundle.appVersion()
+        let lastNewsSeen = ApplicationState.getLastNewsSeen()
         
-        if appVersion.compare(lastNewsSeen, options: .numeric) == .orderedDescending {
+        if "1.0.0".compare("0.9.9", options: .numeric) == .orderedDescending {
```

Comment:
I guess this isn't an purposed change 😄

Reply:
Oh, not it isn't 😓Only for testing purposes, will change it